tox.ini

```diff
@@ -9,18 +9,23 @@ skipsdist = true
 envlist =
     py3
     black
-    flake8 >= 3.7.0
+    flake8
     docs
-    isort >= 5.1.0
+    isort
 
-# Add environment to use the default python3 installation
+# Add environment to use the minmum supported python3 installation
 [testenv:py3]
-basepython = python3
+basepython = python3.8
```

Comment:
I'd revert this and the comment change. `Tox` is not used by CI, and there is not need to force users to use e.g. `3.8` instead of `3.10`.

Reply:
What a pity there is no ability to specify a minimum version for `basepython`. I my case it ran with Python3.7.
I am going to revert this change as the expectation seems to be that developers let `python3` point to a recent version.